components/tools/OmeroPy/src/omero/gateway/__init__.py

```diff
@@ -7823,6 +7823,37 @@ def setInvertedAxis(self, inverted):
         (1, False, True): 'b',  # signed char
         }
 
+    @assert_pixels
+    def getHistogram(self, channels, binCount, globalRange=True,
```

Comment:
Keeping this as `getHistogram()` seems fine to me and is in line with the existing style. While this differs from much of the other Python code, the gateway primarily deals with OMERO objects which do not follow the underscore style anyway.